cpp/src/Ice/SSL/SecureTransportEngine.cpp

```diff
@@ -619,76 +618,35 @@ SecureTransport::SSLEngine::initialize()
 
     if (!certFile.empty())
     {
-        vector<string> files;
-        if (!IceInternal::splitString(certFile, IceInternal::pathsep, files) || files.size() > 2)
```

Comment:
Do we need to document this change?

Reply:
Yes, we will have to update the IceSSL docs. We want to accept a single key/certificate with 3.8 this was already updated for other engines.